lms/djangoapps/ora_staff_grader/tests/test_views.py

```diff
@@ -6,14 +6,19 @@
 
 from uuid import uuid4
 from django.http import QueryDict
-from django.http.response import HttpResponse, HttpResponseForbidden
 from django.urls import reverse
-from opaque_keys.edx.keys import CourseKey
 from rest_framework.test import APITestCase
 from unittest.mock import Mock, patch
 
 from common.djangoapps.student.tests.factories import StaffFactory
-from lms.djangoapps.ora_staff_grader.errors import ERR_BAD_ORA_LOCATION, ERR_LOCK_CONTESTED, ERR_MISSING_PARAM
+from lms.djangoapps.ora_staff_grader.errors import (
+    ERR_BAD_ORA_LOCATION,
+    ERR_GRADE_CONTESTED,
+    ERR_GRADE_SUBMIT,
+    ERR_LOCK_CONTESTED,
+    ERR_MISSING_PARAM,
+    LockContestedError
```

Comment:
Naming inconsistency?

Reply:
The file has both constants and error classes so those names are intentional. Planning to refactor them into separate files to remove this confusion.

Reply:
👍